autoendpoint/src/settings.rs

```diff
@@ -175,13 +177,18 @@ impl Settings {
     // TODO: this should return a Vec<[u8]> so that key formatting errors do not cause
     // false rejections. This is not a problem now since we have access to the source
     // public key, but that may not always be true.
-    pub fn tracking_keys(&self) -> Vec<String> {
+    pub fn tracking_keys(&self) -> Result<Vec<Vec<u8>>, ConfigError> {
```

Comment:
Just this method's doc comment needs updating per the new signature

Reply:
Ah, good catch. Gives me a chance to improve that doc string anyway.